backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/split_shipment/SplitShipmentView_Launcher.java

```diff
@@ -25,6 +27,13 @@ public ProcessPreconditionsResolution checkPreconditionsApplicable(final @NonNul
 			return ProcessPreconditionsResolution.rejectBecauseNotSingleSelection().toInternal();
 		}
 
+		final I_M_ShipmentSchedule shipmentScheduleRecord = context.getSelectedModel(I_M_ShipmentSchedule.class);
```

Comment:
This one is `@deprecated please use {@link #getSingleSelectedRecordId()} and read via DAO/Repository instead`

Reply:
done